client-admin/dev-server.js

```diff
@@ -11,7 +11,8 @@ var request = require("request");
 var app = express();
 var compiler = webpack(config);
 
-var polisConfig = require("./polis.config");
+let POLIS_ROOT = process.env.POLIS_ROOT
```

Comment:
why is this POLIS_ROOT value needed to load a file?

Reply:
This is a legacy from when it seemed important to support running with and without docker (and when the README didn't mention docker). It could be simplified.